habitat/tasks/nav/nav.py

```diff
@@ -398,9 +402,12 @@ def get_observation(
         rotation_world_agent = agent_state.rotation
         rotation_world_start = quaternion_from_coeff(episode.start_rotation)
 
-        return self._quat_to_xy_heading(
-            rotation_world_agent.inverse() * rotation_world_start
-        )
+        if isinstance(rotation_world_agent, quaternion.quaternion):
+            return self._quat_to_xy_heading(
+                rotation_world_agent.inverse() * rotation_world_start
+            )
+        else:
+            raise AttributeError("Agent's rotation was not a quaternion")
```

Comment:
Is this an attribute or a value error?

Reply:
I do not know. I was also hesitating with TypeError. I will change it to Value error.